core/tests/sharing_tests.rs

```diff
@@ -1024,3 +1025,67 @@ fn delete_write_shared_folder() {
     let result = cores[1].delete_file(folder.id);
     assert_matches!(result, Err(UiError(FileDeleteError::InsufficientPermission)));
 }
+
+#[test]
+fn list_metadatas_resolves_links() {
+    let cores = vec![test_core_with_account(), test_core_with_account()];
+    let accounts = cores
+        .iter()
+        .map(|core| core.get_account().unwrap())
+        .collect::<Vec<_>>();
+    let roots = cores
+        .iter()
+        .map(|core| core.get_root().unwrap())
+        .collect::<Vec<_>>();
+
+    let folder = cores[0]
```

Comment:
This is created only for the `id` it seems? If so, could just grab id as you do with `link_id` below.

Reply:
[c830e43](https://github.com/lockbook/lockbook/pull/1322/commits/c830e43226c06391e2d4c7b03b555ec0630620b5)